aptos-move/move-examples/cli-e2e-tests/common/sources/cli_e2e_tests.move

```diff
@@ -267,6 +267,21 @@ module addr::cli_e2e_tests {
         (num64, num128, num256)
     }
 
+    #[view]
+    public fun test_vector(
+        input_u8: vector<u8>,
+        input_u16: vector<u16>,
+        input_u32: vector<u32>,
+        input_u64: vector<u64>,
+        input_u128: vector<u128>,
+        input_u256: vector<u256>,
+        input_addr: vector<address>,
+        input_bool: vector<bool>,
+        input_string: vector<String>,
+    ): (vector<u8>, vector<u16>, vector<u32>, vector<u64>, vector<u128>, vector<u256>, vector<address>, vector<bool>, vector<String>) {
+        (input_u8, input_u16, input_u32, input_u64, input_u128, input_u256, input_addr, input_bool, input_string)
+    }
```

Comment:
What about nested nested vectors?

Basically what I'm seeing right now is that the code that was previously here is basically being added in for single nested vectors

Reply:
Good question. I was trying to write a recursive func to handle all depth level but failed lol

Since there isn't that much demand on nested vector for u64 - u256. Lets get these into our next release and bandage it. I will follow up with a nested vector fix.